### DISCUSSION TITLE
Cara membuat bubble charts menggunakan Matpotlib

### DISCUSSION THREAD

**Discussion by namiraprita** [teamalgoritma/community#313] [Q&A] [answered]
Hi Algoritma Team,

Saya ingin membuat data "employees_total"  menjadi scatter plot by "company name" dan size yang diadjust sesuai dengan banyaknya jumlah employee - makin besar jumlahnya, bubble charts semakin besar juga. bagaimana ya caranya kalo pakai groupby sebelumnya? karena column namenya jadi kosong untuk jumlah employeenya. terima kasih

![Screen Shot 2022-09-20 at 16 20 46](https://user-images.githubusercontent.com/107253301/191357218-1ab89202-9b58-485c-b660-c1e27c37ad51.png)

**Comment by kevbow** [comment, marked as answer]
Halo @namiraprita,

Apakah sudah mencoba menggunakan parameter size menggunakan fungsi scatterplot dari package seaborn? 
Berikut salah satu [referensi ](https://www.python-graph-gallery.com/bubble-plot/)yang dapat digunakan.
Untuk data x dan y nya juga apabila memungkinkan mohon untuk ditampilkan agar kami terbayang bentuknya. 
Mohon maaf keterlambatannya dan terima kasih atas pengertiannya.
